apps/slackbot/utils.py

```diff
@@ -110,8 +112,10 @@ def count_string_tokens(string: str, model_name: str) -> int:
     return len(encoding.encode(string))
 
 
-def chunk_and_summarize(text_data: str, question: str, open_ai_key: str, link: str):
-    llm = OpenAI(temperature=0.9, openai_api_key=open_ai_key)
+def chunk_and_summarize(text_data: str,  question: str, open_ai_key: str, link: str, team_id: str = None, user_id: str = None):
+    llm = SherpaOpenAI(
+        temperature=0.9, openai_api_key=open_ai_key, user_id=user_id, team_id=team_id)
```

Comment:
Here we should use the configured temperature

Reply:
fixed it check it out

Reply:
I forgot to change it inside the utility that was the case

Reply:
its changed now

Reply:
its changed now @20001LastOrder